app/src/main/java/com/example/sharingang/NewItemFragment.kt

```diff
@@ -30,8 +30,15 @@ class NewItemFragment : Fragment() {
     // Allows the cancellation of a location request if, for example, the user exists the activity
     private var cancellationTokenSource = CancellationTokenSource()
     private val requestPermissionLauncher = requestPermissionLauncher(this) {
-        doOrGetPermission(this, Manifest.permission.ACCESS_FINE_LOCATION,
-            { consumeLocation(fusedLocationCreate, cancellationTokenSource) { updateLocation(it) } }, null)
+        doOrGetPermission(
+            this, Manifest.permission.ACCESS_FINE_LOCATION,
+            {
+                consumeLocation(
+                    fusedLocationCreate,
+                    cancellationTokenSource
+                ) { updateLocation(it) }
+            }, null
+        )

```

Comment:
I had to reduce the length of this part (into two lines) to reduce the amount of uncovered code, you might want to put it back as it was if you can't reach the 80% coverage